web/src/main/webapp/WEB-INF/pages/candidate_create.jsp

```diff
@@ -198,6 +204,71 @@
                 </div>
             </div>
         </div>
+
+        <c:if test="${cv_status eq true}">
+        <c:set var="candidateCvPath" value="${candidate.cvPath}" />
+        <c:set var="cvPathArray" value="${fn:split(candidateCvPath, '\\\\')}" />
+        <c:set var="cvName" value="${cvPathArray[fn:length(cvPathArray)-1]}" />
+        <label class="control-label col-lg-2 col-md-2 col-sm-2 text-right">
+          CV : <a href="<c:url value='/secure/candidate/fileDownload/${candidateId}' />">
+          <c:out value="${cvName}">
+        </c:out>
+        </a>
+        </label>
+        </c:if>
+
+        <c:if test="${not empty candidateId}">
+        <form:form method="POST" action="${fileUpload}/${candidateId}" enctype="multipart/form-data">
+
+                            <c:if test="${cv_status eq false}">
+                        		 <h4><spring:message code="please.upload.cv"/></h4>
+                        		<input type="file" name="file" />
+                        		<button type="submit" class="btn btn-primary"><spring:message code="upload.button"/></button>
+                            </c:if>
+
+                     	 <c:if test="${not empty validationSuccessKey}">
+                             <div id="globalMessage" class="alert alert-success" role="alert">
+                                 <span class="glyphicon glyphicon-ok" aria-hidden="false"></span>
+                                    <spring:message code="${validationSuccessKey}"/>
+                             </div>
+                         </c:if>
+
+                         <c:if test="${not empty validationErrorKey}">
+                            <div id="globalMessage" class="alert alert-danger" role="alert">
+                                <span class="glyphicon glyphicon-exclamation-sign" aria-hidden="false"></span>
+                                    <spring:message code="${validationErrorKey}"/>
+                            </div>
+                         </c:if>
+        </form:form>
+        </c:if>
+
+        <c:if test="${empty candidateId}">
+        <form:form method="POST" action="${fileUpload}" enctype="multipart/form-data">
+
+                    <c:if test="${cv_status eq false}">
```

Comment:
Formatting errors. Maybe tabs are used mixed with spaces. Should be only spaces.